packages/amplify-graphql-api-construct-tests/package.json

```diff
@@ -98,7 +98,10 @@
     },
     "moduleNameMapper": {
       "axios": "axios/dist/node/axios.cjs"
-    }
+    },
+    "coveragePathIgnorePatterns": [
+      "/__tests__/"
```

Comment:
Redundant w/L63 above. I think having a single way of doing exclusions would be nice, as long as it results in the desired behavior, so I'd recommend adding L62-L64 to the `coveragePathIgnorePatterns` for all of them. Even better: is there a way to configure this centrally rather than having to specify it across every file? What is the interplay between these package-scoped `package.json` declarations and the top level ones in `/jest.config.js`?

Reply:
There is a `projects` [config in jest for monorepos](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig). This looks like it doesn't work well with lerna. The way we have it setup now allows us to run tests for packages for tests individually. Using a single jest config with `projects` would run all of the tests at once.

The best way to share config across the packages would be to create a `jest.config.base` file at the root of the project and have all the packages extend the base config. I think that should be taken on as a separate change though.